src/MatrixClient.ts

```diff
@@ -1421,6 +1422,75 @@ export class MatrixClient extends EventEmitter {
         return result;
     }
 
+    /**
+     * Creates a Space room.
+     * @param {SpaceCreateOptions} opts The creation options.
+     * @returns {Promise<Space>} Resolves to the created space.
+     */
+    public async createSpace(opts: SpaceCreateOptions): Promise<Space> {
+        const roomCreateOpts = {
+            name: opts.name,
+            topic: opts.topic || "",
+            preset: opts.isPublic ? 'public_chat' : 'private_chat',
+            room_alias_name: opts.localpart,
+            initial_state: [
+                {
+                    type: "m.room.history_visibility",
+                    state_key: "",
+                    content: {
+                        history_visibility: opts.isPublic ? 'world_readable' : 'shared',
+                    },
+                },
+            ] as unknown[],
+            creation_content: {
+                type: "m.space",
+            },
+            invite: opts.invites || [],
+            power_level_content_override: {
+                ban: 100,
+                events_default: 50,
```

Comment:
Currently looking into spaces support in the matrix-rust-sdk and wondered: [MSC1772](https://github.com/matrix-org/matrix-doc/blob/master/proposals/1772-groups-as-rooms.md) states

> Space-rooms should be created with a power level for `events_default` of 100

Is 50 used here for backwards-compatibility or was there another reason?